annotations/mockery/cmd/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"fmt"
+	"net/http"
+
+	ann "github.com/Sotera/go_watchman/annotations"
+	mock "github.com/Sotera/go_watchman/annotations/mockery"
+	"github.com/Sotera/go_watchman/loogo"
+	"gopkg.in/gin-gonic/gin.v1"
+)
+
+var eventsAPIRoot = os.Getenv("EVENTS_API_ROOT")
+
+func main() {
+
+	if eventsAPIRoot == "" {
+		eventsAPIRoot = "http://localhost:3000/api/events"
+	}
+
+	r := gin.Default()
+
+	parser := &loogo.HTTPRequestParser{
+		Client: &loogo.HTTPClient{},
+	}
+	mocker := mock.Mockery{
+		Parser: parser,
+	}
+
+	annotationsGroup := r.Group("/annotations")
+	{
+		annotationsGroup.GET("/refid/:refid", func(c *gin.Context) {
+			// refid := c.Param("refid")
```

Comment:
@justinlueders do we need to use this in the mockery somehow? or just a pass thru?